website/common/locales/en/character.json

```diff
@@ -214,16 +214,12 @@
   "editProfile": "Edit Profile",
   "challengesWon": "Challenges Won",
   "questsCompleted": "Quests Completed",
-  "equipment": "Equipment",
-  "costume": "Costume",
   "headAccess": "Head Access.",
   "backAccess": "Back Access.",
   "bodyAccess": "Body Access.",
   "mainHand": "Main-Hand",
   "offHand": "Off-Hand",
-  "level": "Level",
-  "allocated": "Allocated",
-  "buffs": "Buffs",
   "statPoints": "Stat Points",
+  "pointsAvailable": "Points Available",
```

Comment:
is this string used anywhere?

Reply:
That was an old Stat Points title, removed in https://github.com/HabitRPG/habitica/pull/10543/files and no longer needed.

Reply:
Cool, I'll remove it.